_posts/2023-01-05-exploring-process-automation-opportunities.md

```diff
@@ -0,0 +1,60 @@
+
+## Process automation
+
+Automation in general aims to decrease the necessity of human interaction in different processes. Manufacturing various products has been a target of automation for decades. As technological development has progressed is has become possible to automate repetitive tasks in modern office work as well. Repetitive tasks that follow certain rules and have a significant risk of human error are usually very suitable for process automation. The global business process automation market is expected [to grow up to $19,4 billion by 2026](https://www.marketsandmarkets.com/Market-Reports/business-process-automation-market-197532385.html)
+
+Process automation has not yet become a popular research subject, which is why there are still common misconceptions regarding this subject. Most notably, researchers often refer robotic process automation (RPA) as the one and only form of process automation and state that these two terms as the same. RPA is just one form of process automation since any form of development which reduces the necessity of manual labor can be considered as process automation. This includes system integrations, process planning and software development. RPA has been the most studied form of process automation for few years, which is why it has gained majority of the focus. 
+
+## RPA
+
+Robotic Process Automation, or RPA, has nothing to do with mechanical robots used in different factories, but instead is a purely software-based solution, designed to mimic different tasks performed by human employees. Developing an RPA solution often starts with determining all the steps a human worker performs in a certain process, including every input and click of a button. All the potential exceptions and errors should also be documented. After scripting all the steps, the software robot can mimic the described process.
+
+The reason RPA solutions can be more productive than human workers in certain processes it that RPA solutions are faster, and they do not make any humane errors, like typos or misinterpretations. From developers’ point of view, creating RPA solutions can be fast and low effort since the solution is implementable to use existing systems without the necessity to alter said systems in any way. The RPA solution can do all the same tasks as a human worker, so access to required systems can be obtained in a comparable manner, as with human employee.
```

Comment:
can we have some example illustrations of RPA? to show what kinds of things can be done

Reply:
Added illustration for potential use cases